tests/vision/classification/test_data.py

```diff
@@ -96,6 +96,31 @@ def test_from_filepaths(tmpdir):
     assert labels.shape == (1, )
 
 
+def test_from_filepaths_visualise(tmpdir):
+    tmpdir = Path(tmpdir)
+
+    (tmpdir / "a").mkdir()
+    (tmpdir / "b").mkdir()
+    _rand_image().save(tmpdir / "a" / "a_1.png")
+    _rand_image().save(tmpdir / "a" / "a_2.png")
+
+    _rand_image().save(tmpdir / "b" / "a_1.png")
+    _rand_image().save(tmpdir / "b" / "a_2.png")
+
+    data_viz = ImageClassificationDataVisualizer.from_filepaths(
+        train_filepaths=[tmpdir / "a", tmpdir / "b"],
+        train_labels=[0, 1],
+        val_filepaths=[tmpdir / "a", tmpdir / "b"],
+        val_labels=[0, 1],
+        test_filepaths=[tmpdir / "a", tmpdir / "b"],
+        test_labels=[0, 1],
+        batch_size=2,
+    )
+    data_viz.show_train_batch()
+    data_viz.show_val_batch()
+    data_viz.show_test_batch()
```

Comment:
we need to decide how we validate this functionality during tests since it involves matplotlib visualization

Reply:
You could use: https://matplotlib.org/stable/devel/testing.html